packages/api/src/controllers/farmExpenseTypeController.js

```diff
@@ -38,6 +38,7 @@ const farmExpenseTypeController = {
           } else {
             // if its deleted, them make it active
             record.deleted = false;
+            record.custom_description = data.custom_description ?? null;
```

Comment:
Good spotting!

Yeah I don't like how this is handled either with regards to un-retiring. But basically the logic is.. if they are un-retiring a type using the same name -- description supplied may be different so that should update. -- Really don't love it but if someone is making a new type that would normally succeed if there is a duplicate name than it should show all the inputted data. 

Any empty strings are definitely a mistake. And this should be undefined so as not to mess with the existing description. I just forgot to apply it to all the instances.

Reply:
Don't worry about un-retiring via add as we are removing that before release ([Jira ticket](https://lite-farm.atlassian.net/browse/LF-3666)

Reply:
I like to keep things working in case it gets moved to nice to have haha.

But nullish coalescing got me in trouble here. Empty strings should not be permitted and I forget it works differently than ternary or regular logic.